### PR TITLE
Change units of time

### PR DESCRIPTION
**Pull request by Phoenix-Starlight**
One of the PRs (hopefully just one!) to replace units of time to something more flexible and accurate.

### CODE REVIEW

File: pages/Animal-Growth-Accelerator.md [hunk 1]
```diff
@@ -6,5 +6,5 @@ It runs on [Organic Food](https://github.com/Slimefun/Slimefun4/wiki/Miscellaneo
 
 There is only one tier of the Animal Growth Accelerator and it has the following properties:  
 
-**Power Usage**: 28 J/s  
+**Power Usage**: 28 J/SF TickF tick
```
Comment (svr333) [inline pages/Animal-Growth-Accelerator.md:9]:
```suggestion
**Power Usage**: 28 J/SF Tick
```

**Review by svr333 — DISMISSED** [review]
Slimefun so the abbreviation would be Sf :)

File: pages/Animal-Growth-Accelerator.md (hunk 1 above)
Comment (Phoenix-Starlight) [reply pages/Animal-Growth-Accelerator.md:9]: Hm- sed didn't catch that? Oh noes!

**Review by svr333 — CHANGES_REQUESTED** [review]
all the tables are gonna need to be reformatted too

File: pages/Freezer.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ This can be a very useful machine, for example, to create blue ice in the long-r
 
 ## Freezer Tiers
 
-| Tier | Power  | Speed Multiplier | Buffer |
-| :--: | :----: | :--------------: | :----: |
-| I    | 18 J/s | 1.0x             | 256 J  |
-| II   | 30 J/s | 2.0x             | 256 J  |
+| Tier | Power        | Speed Multiplier | Buffer |
+| :--: | :----:       | :--------------: | :----: |
```
Comment (svr333) [inline pages/Freezer.md:7]:
```suggestion
| Tier |     Power    | Speed Multiplier | Buffer |
| :--: | :----------: | :--------------: | :----: |
```

File: pages/GPS-Transmitter.md [hunk 3]
```diff
@@ -9,9 +9,9 @@ The GPS Transmitter adds GPS network complexity equal to `TierModifier * Height
...
 The table below describes differences between tiers of Transmitters.
 
-| Tier | Buffer | Consumption | Tier Modifier | Tier Bonus |
-| ---- | ------ | ----------- | ------------- | ---------- |
-| GPS Transmitter | 16 J | 2 J/s | 1 | 0 |
-| Advanced GPS Transmitter | 64 J | 6 J/s | 4 | 100 |
-| Carbonado GPS Transmitter | 256 J | 22 J/s | 16 | 500 |
-| Energized GPS Transmitter | 1024 J | 92 J/s | 64 | 600 |
+| Tier                      | Buffer | Consumption  | Tier Modifier | Tier Bonus |
+| ----                      | ------ | -----------  | ------------- | ---------- |
```
Comment (svr333) [inline pages/GPS-Transmitter.md:13]:
```suggestion
| Tier                      | Buffer | Consumption  | Tier Modifier | Tier Bonus |
| ------------------------- | ------ | ------------ | ------------- | ---------- |
```

File: pages/Heated-Pressure-Chamber.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ It is a more robust and more powerful version of the [Pressure Chamber](https://
 
 ## Heated Pressure Chamber Tiers
 
-| Tier | Power  | Speed Multiplier |
-| :--: | :----: | :--------------: |
-| I    | 10 J/s | 1.0x             |
-| II   | 44 J/s | 5.0x             |
\ No newline at end of file
+| Tier |    Power     | Speed Multiplier |
+| :--: |    :----:    | :--------------: |
```
Comment (svr333) [inline pages/Heated-Pressure-Chamber.md:8]:
```suggestion
| Tier |    Power     | Speed Multiplier |
| :--: | :----------: | :--------------: |
```

File: pages/Solar-Generator.md [hunk 5]
```diff
@@ -15,9 +15,9 @@ Solar generators do not have a buffer, so any unused energy produced by them wil
...
 The table below compares the energy production of solar generators.
 
-| Tier                      | Energy                           |
-| :------------------------ | :------------------------------: |
-| Solar Generator           |              4 J/s               |
-| Advanced Solar Generator  |              16 J/s              |
-| Carbonado Solar Generator |              64 J/s              |
-| Energized Solar Generator | 256 J/s (Day)<br>128 J/s (Night) |
+| Tier                      | Energy                                       |
+| :------------------------ |       :------------------------------:       |
```
Comment (svr333) [inline pages/Solar-Generator.md:19]:
```suggestion
| Tier                      | Energy                                       |
| :------------------------ | :------------------------------------------: |
```

File: pages/Reactors.md [hunk 6]
```diff
@@ -7,10 +7,10 @@ It has to be placed exactly 3 blocks above the reactor and it can be used to int
 This provides an easy way to add Fuel/Coolant or take some of the byproduct.
 
 ## Reactor Types
-| Type                | Power   | Buffer | Fuel                                | Cooling                  | Side Effects                 |
-| :------------------ | :-----: | :----: | ----------------------------------- | ------------------------ | ---------------------------- |
-| Nuclear Reactor     | 500 J/s |  1.0x  | Neptunium, Uranium, Boosted Uranium | Reactor Coolant Cells    | N/A                          |
-| Nether Star Reactor | 40 J/s  |  3.0x  | Nether Stars                        | Nether Ice Coolant Cells | Nearby Entities get Withered |
+| Type                 | Power         | Buffer | Fuel                                | Cooling                  | Side Effects                 |
+| :------------------  |    :-----:    | :----: | ----------------------------------- | ------------------------ | ---------------------------- |
```
Comment (svr333) [inline pages/Reactors.md:11]:
```suggestion
| Type                 | Power         | Buffer | Fuel                                | Cooling                  | Side Effects                 |
| :------------------  | :------------: | :----: | ----------------------------------- | ------------------------ | ---------------------------- |
```

File: pages/Lava-Generator.md [hunk 7]
```diff
@@ -9,10 +9,10 @@ A lava generator must be placed within 7 blocks of an [Energy Regulator](https:/
 ## Tiers
 There are currently two tiers of lava generators.
 
-| Tier | Energy | Buffer |
-| :--: | :----: | :----: |
-| I    | 20 J/s | 512 J  |
-| II   | 40 J/s | 1024 J |
+| Tier |    Energy    | Buffer |
+| :--: |    :----:    | :----: |
```
Comment (svr333) [inline pages/Lava-Generator.md:13]:
```suggestion
| Tier |    Energy    | Buffer |
| :--: | :----------: | :----: |
```

**Review by svr333 — CHANGES_REQUESTED** [review]

**Comment by Phoenix-Starlight** [comment]
Ah I see now!

**Comment by Phoenix-Starlight** [comment]
Fixes #198 (just so this automatically closes)

**Comment by Phoenix-Starlight** [comment]
But.. it's not changed here? If you mean the other one, I reverted that.

**Comment by svr333** [comment]
yeah but you mentioned that this PR fixes #198 which is unrelated

**Comment by Phoenix-Starlight** [comment]
Oh. yeah.

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**